packages/contracts/foundry.toml

```diff
@@ -39,3 +39,6 @@ targets = [
 
 [profile.docs]
 src = 'src/dollar'
+
+[profile.intense.fuzz]
+runs = 10000
```

Comment:
If this only takes 2-3 minutes, why not crank this up to something excessive? Perhaps something that will get us close to the GitHub Actions max runner runtime of 6 hours?

Reply:
Hi @pavlovcik 1 000 000 took ~20 minutes on my quite powerful laptop. This is due to fuzzer logic and increasing corpus which stores coverage.

But it may not make sense to set this to such large value anyway, see https://consensys.io/diligence/blog/2023/04/benchmarking-smart-contract-fuzzers/  , at some point fuzzer corpus saturates and it does not find more issues. Imho we could try to set to 100 000 , which currently takes ~7 minutes . Please let me know and I will update the value to the one you suggest.

Reply:
In the article they said they ran each fuzzing "campaign" for eight hours so based on that we should aim for the full six hours on the GitHub runner. 

It also mentions caching so perhaps we can preserve the cache by uploading the artifact, and then downloading it on the next run?

Reply:
> would just consume resources 

GitHub Actions is a free resource which is why we should take advantage of it.

Reply:
@pavlovcik ok, sure, shall I set the value to 100 000 000 runs ?

Reply:
done